src/Features/Blockcore.Features.BlockStore/Blockcore.Features.BlockStore.csproj

```diff
@@ -20,6 +20,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\External\NBitcoin\NBitcoin.csproj" />
     <ProjectReference Include="..\..\Blockcore\Blockcore.csproj" />
+    <ProjectReference Include="..\Blockcore.Features.RPC\Blockcore.Features.RPC.csproj" />
```

Comment:
This is wrong we should not be adding a dependency to the RPC project, why do you need it what dependency are you missing? (maybe we can sort it out differently)

Reply:
ah ok I see we are actually sharing the RPC projects around in other projects so fine